karate-core/src/main/java/com/intuit/karate/Main.java

```diff
@@ -132,6 +132,9 @@ public class Main implements Callable<Void> {
     @Option(names = {"-H", "--hook"}, split = ",", description = "class name of a RuntimeHook (or RuntimeHookFactory) to add")
     List<String> hookFactoryClassNames;
 
+    @Option(names = {"--keep-original-headers"}, description = "keeping original headers given in the mock scenario or configure")
```

Comment:
Allow the mock server to enable or disable the headers format, as not everyone may need to use HTTP/1 headers like I do 👍